store/generated_key/eigenda/eigenda.go

```diff
@@ -27,7 +28,17 @@ type StoreConfig struct {
 	// total duration time that client waits for blob to confirm
 	StatusQueryTimeout time.Duration
 
-	// number of times to retry eigenda blob dispersals
+	// PutRetries specifies the total number of times to try dispersing a blob.
+	// This value comes from utils.MapPutRetries() and is passed directly to retry-go.
+	//
+	// IMPORTANT: In retry-go, this parameter means:
+	// - Value of 0: Try indefinitely until success
+	// - Value of 1: Try once with no retries
+	// - Value > 1: Try up to this many times total (first try + up to N-1 retries)
+	//
+	// Users should NEVER set this field directly. Always use NewStoreConfig()
+	// which correctly maps from the user-friendly int values to the internal uint values
+	// required by the retry-go library.
 	PutRetries uint
```

Comment:
don't really like this. I would prefer if all our configs were self-consistent (had same meaning) and we called MapPutRetries at the retry call site only (adds an inconsequential amount of processing overhead)